libraries/helpers.rb

```diff
@@ -28,7 +28,8 @@ def psql_command_string(new_resource, query, grep_for: nil, value_only: false)
       cmd << " --host #{new_resource.host}" if new_resource.host
       cmd << " --port #{new_resource.port}" if new_resource.port
       cmd << ' --tuples-only'               if value_only
-      cmd << " | grep #{grep_for}"          if grep_for
+      cmd << ' --no-psqlrc'
```

Comment:
Thinking about this more, I think it might be a good idea to make this a new parameter called `psqlrc` which is enabled by default (which mimics the current behavior), but folks like you can disable it.

I can just see this impacting folks who are expecting this to work like it has been.

Reply:
There was an attempt to get a boolean value from node attributes such as `node['postgresql']['no_psqlrc']` as to have a way to change the behaviour of the already defined helpers, but I did not succeed... 😞

Reply:
This would be in the resource and not done via attributes. We're trying to move away from attributes towards more resource driven cookbooks so we don't run into issues.  Would you like some help or give an example of how to do this?

Reply:
That would be great! 🙇

Reply:
Here's a diff with an example. You'd want to add this property to every resource that also has a `host` property.
```diff
diff --git a/libraries/helpers.rb b/libraries/helpers.rb
index 911a094..d473ed0 100644
--- a/libraries/helpers.rb
+++ b/libraries/helpers.rb
@@ -21,14 +21,14 @@ module PostgresqlCookbook
 
     require 'securerandom'
 
-    def psql_command_string(new_resource, query, grep_for: nil, value_only: false, no_psqlrc: false)
+    def psql_command_string(new_resource, query, grep_for: nil, value_only: false)
       cmd = %(/usr/bin/psql -c "#{query}")
       cmd << " -d #{new_resource.database}" if new_resource.database
       cmd << " -U #{new_resource.user}"     if new_resource.user
       cmd << " --host #{new_resource.host}" if new_resource.host
       cmd << " --port #{new_resource.port}" if new_resource.port
       cmd << ' --tuples-only'               if value_only
-      cmd << ' --no-psqlrc'                 if no_psqlrc
+      cmd << ' --no-psqlrc'                 unless new_resource.psqlrc
       cmd << " | grep #{grep_for}"          if grep_for
       cmd
     end
diff --git a/resources/database.rb b/resources/database.rb
index c5307fb..a17d5c3 100644
--- a/resources/database.rb
+++ b/resources/database.rb
@@ -26,6 +26,7 @@ property :user,     String, default: 'postgres'
 property :database, String, name_property: true
 property :host,     String
 property :port,     Integer, default: 5432
+property :psqlrc,   [true,false], default: true
 
 action :create do
   createdb = 'createdb'
```

Reply:
🙇 🙇 🙇 I have used your example and fixed my PR accordingly.